templates/index.html

```diff
@@ -6,29 +6,29 @@
       <div class="row mt-3">
         <div class="col-md-4 mx-auto">
           <div class="form-group">
-            <label class="heading" id="label" for="email">Email *</label>
+            <label class="heading" id="label" for="email">Email *Required</label>
             <div class="input-group margin-bottom-sm">
               <span class="input-group-addon"><i class="fa fa-envelope-o fa-fw"></i></span>
-              <input autofocus="" class="form-control" placeholder="Enter email" name="email" title="Email for sending the ISO"
+              <input autofocus="" class="form-control" placeholder="Enter email" name="email" title="Email for the ISO"
                 required="" type="email" pattern="^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)+$"
                 required /></div>
-            <p id="scroll-to">We send you the link to the ready ISO per email.</p>
+            <p id="scroll-to">We will send the ready ISO file to the e-mail you provide</p>
             <br>
           </div>
           <div class="form-group">
-            <label class="heading" for="TRAVIS_TAG">Name of your Distribution *</label>
+            <label class="heading" for="TRAVIS_TAG">Name of your Distribution *Required</label>
             <div class="input-group margin-bottom-sm">
               <span class="input-group-addon"><i class="fa fa-address-book fa-fw"></i></span>
               <input pattern="[A-Za-z0-9\s]{1,20}" class="form-control" placeholder="Enter name" name="TRAVIS_TAG"
-                required="" type="text" data-toggle="tooltip" title="Only alphanumeric characters allowed"></div>
+                required="" type="text" data-toggle="tooltip" title="Only Alphanumeric Characters Allowed"></div>
             <p id="scroll-to">Name of your custom distribution</p>
           </div>
           <br>
           <div class="form-group">
             <label class="heading" for="event_url">Browser Home Page </label>
             <div class="input-group margin-bottom-sm">
               <span class="input-group-addon"><i class="fa fa-window-maximize fa-fw"></i></span>
-              <input pattern="https?://.+" class="form-control" placeholder="Enter link" name="event_url" title="only url is allowed"
+              <input pattern="" class="form-control" placeholder="Enter link" name="event_url" title="Only URL is allowed"
```

Comment:
I tried using the site before editing the code. I entered www.google.com in and it told me that it is not valid. I think the validation might a little too strict.  Not a lot of people will write https://www.google.com when they want to go to google.

Reply:
For that, you can edit the pattern but atleast there should be a validation 
You should also provide a test link; for the same so that we can test

Reply:
Hi, sorry for this but I am really not familiar with html validation techniques. I think using a validation where:
websitename + '.' + suffix should be the minimum validation

Reply:
Or maybe you can just put https:// or http:// before the field for people to fill in, such that when the person types something in, it will automatically put http:// in

Reply:
You can always search over the internet for suitable patterns 
For the second option, I dont know if it will be necessary maybe @tabesin, @meets2tarun  can help

Reply:
Personally I think the second option is more sensible, however I really don't know how to do it. Maybe you can do a check which is more background, which is something like the site will ping that site with packets of 32 bits (same as cmd ping commands), if there are any replies then that site can be used, however if it does not reply then I think maybe the user needs to re-input the site

Reply:
Wait for @tabesin  to review

Reply:
This is not a UI for the average joe. I think http validation is fine, esp when you provide an example. In any case we do not have to get into this at this stage for a mvp.